unstract/sdk1/tests/test_prompt.py

```
@@ -0,0 +1,154 @@
+            prompt_port="3003",
+            is_public_call=False,
+            request_id="test-request-id",
+        )
+
+        payload = {"prompt": "test"}
+
+        with patch("requests.post") as mock_post:
+            mock_post.side_effect = ConnectionError("Persistent failure")
+
+            # Exception handled by decorator
+            with pytest.raises(Exception):
```

Ruff (0.13.3)

Do not assert blind exception: `Exception`

(B017)